group/models.py

```diff
@@ -142,10 +142,18 @@ class MeetingMinutes(AbstractReport):
     """
     Meeting minutes content type.
     """
-    panels = AbstractReport.panels
+    date = models.DateField(blank=False)
+    summary = models.TextField(null=False, blank=False)
+
+    panels = [
+        FieldPanel('date'),
+        FieldPanel('summary'),
+        PageChooserPanel('link_page'),
+    ]
 
     class Meta:
         abstract = True
+        ordering = ['-date']
```

Comment:
I'll trust you on this one but you might want to check this on `nest` if you're not certain about it. It will effect ordering, I just don't remember exactly how. Assuming you've looked at this closely, disregard my paranoia.

Reply:
This was part of `AbstractReport`, but since it is inherited, I removed it.